app/components/ui/Navigation.module.scss

```diff
@@ -172,3 +178,29 @@
     font-style: normal;
   }
 }
+
+.buttonLink {
+  appearance: button;
+  text-decoration: none;
+  color: white;
+  background-color: $color-brand;
+  border-radius: 3px;
+  font-size: 16px;
+  font-weight: 600;
+  height: 60px;
+  width: 200px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
You may not need these if we decide to using both vertical and horizontal padding, since that will naturally center the text in both dimensions. If we do still want to use flexbox, then we should at least make this an `inline-flex` so that it behaves like a normal button when inside of [normal flow](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flow_Layout) (e.g. a non-flexbox, non-grid, non-table element).

Reply:
I'll try to get it centered with padding 👍